app/src/components/NavBar.vue

```diff
@@ -30,7 +28,7 @@
           class="mt-3 dropdown-content z-[1] menu p-2 shadow bg-white rounded-box w-48"
         >
           <li v-if="withdrawLoading"><a href="#">Processing...</a></li>
-          <li v-else><a @click="withdraw()">Withdraw Tips</a></li>
+          <li v-else><a @click="$emit('withdraw')">Withdraw Tips</a></li>
```

Comment:
```suggestion
          <li v-else><a @click="emits('withdraw')">Withdraw Tips</a></li>
```
I think its better to define it this way since we have defined the emits variable.